demo/src/screens/foundationScreens/DarkModeScreen.tsx

```diff
@@ -1,22 +1,46 @@
 import React, {Component} from 'react';
 import {StyleSheet} from 'react-native';
-import {View, Text, Constants} from 'react-native-ui-lib';
+import {View, Text, Colors, Constants, SegmentedControl, SegmentedControlItemProps} from 'react-native-ui-lib';
+
+const SCHEME_TYPES = [
+  {label: 'device (default)', value: 'default'},
+  {label: 'dark', value: 'dark'},
+  {label: 'light', value: 'light'}
+];
+
+const DEVICE_DARK_MODE_MESSAGE = Constants.isIOS
+  ? 'Change to dark mode in simulator by pressing Cmd+Shift+A'
+  : 'Change to dark mode';
+const MANUAL_DARK_MODE_MESSAGE = 'Setting the scheme manually will ignore device settings';
 
 class DarkModeScreen extends Component {
-  state = {};
+  state = {
+    selectedSchemeType: 'default'
+  };
+
+  changeSchemeType = (index: number) => {
+    this.setState({selectedSchemeType: SCHEME_TYPES[index].value});
+    Colors.setScheme(SCHEME_TYPES[index].value);
```

Comment:
There's a ts error here - it requires declaring the type at the `SCHEME_TYPES` array

Reply:
Fixed